src/js/BookReader.js

```diff
@@ -343,7 +344,13 @@ BookReader.prototype.setup = function(options) {
   this.getPageProp = options.getPageProp || BookReader.prototype.getPageProp;
   this.getSpreadIndices = options.getSpreadIndices || BookReader.prototype.getSpreadIndices;
   this.leafNumToIndex = options.leafNumToIndex || BookReader.prototype.leafNumToIndex;
+
+  /** @type {{[name: string]: JQuery}} */
   this.refs = {};
+  
+  this.components = {
```

Comment:
Will this be the place to add new components?
What do components consist of?
Do you think we need a lil note above this to instruct future people to register components here?